src/agent/configuration_parser/include/configuration_github.hpp

```diff
@@ -0,0 +1,75 @@
+#include <iostream>
+
+
+namespace configurationGithub
+{
+    const bool DEFAULT_ENABLED = true;
+    const std::string DEFAULT_INTERVAL = "10m";
+    const std::string DEFAULT_TIME_DELAY = "30s";
+    const std::string DEFAULT_CURL_MAX_SIZE = "1M";
+    const bool DEFAULT_ONLY_FEATURE_EVENTS = true;
+    const std::string DEFAULT_EVENT_TYPE = "all";
+
+    class GithubConfigApiAuth
+    {
+        private:
+            std::string orgName;
+            std::string apiToken;
+        public:
+            GithubConfigApiAuth(){};
```

Comment:
There's no no need for a semicolon after the closing } of a function definition or a control structure.